neural_compressor/compression/pruner/model_slim/pattern_analyzer.py

```diff
@@ -183,44 +183,56 @@ def generate_static_graph_with_dummyinput(self):
                 torch.ones([1, 80, 3000]),
                 torch.ones([1, 448], dtype=torch.int64)

```

Comment:
it is quite tricky , is there any chance to polish it

Reply:
This will not be called if we provide an external datasets for whisper model. However, it is still hard to analyze whisper's dummy input.